bin/fin

```diff
@@ -5054,7 +5041,7 @@ update ()
 	if is_linux; then
 		# Override docker to run it via sudo
 		docker () {
-			sudo "$(which docker)" "$@"
+			sudo command docker "$@"
```

Comment:
I don't think this works. I get this when I try this:

```
lpeabody@pop-os:~/Projects/eqrx-intranet (master)$ sudo command docker
sudo: command: command not found
```

Reply:
Googling, I found https://forums.opensuse.org/showthread.php/439150-sudo-lt-command-gt-command-not-found?s=62a70e125aab2af9c711eac9a69f792b&p=2167748#post2167748:

`sudo -i command docker` works for me, where as `sudo command docker` fails. I don't know if adding `-i` causes any other funny business...

Reply:
Interesting... What's OS/shell? Does this work? `command -v ls`

Reply:
Linux Pop!_OS 20.04
Linux pop-os 5.8.0-7642-generic #47~1614007149~20.04~82fb226-Ubuntu SMP Tue Feb 23 02:56:27 UTC  x86_64 x86_64 x86_64 GNU/Linux

```
lpeabody@pop-os:~/Projects/baxter-myclaria (feature/myclaria-site-profile)$ bash --version
bash --version
GNU bash, version 5.0.17(1)-release (x86_64-pc-linux-gnu)
```

```
lpeabody@pop-os:~/Projects/docksal/service-cli (feature/composer-2-14)$ command ls
7.3  7.4  8.0  code-server  LICENSE  Makefile  README.md  tests
lpeabody@pop-os:~/Projects/docksal/service-cli (feature/composer-2-14)$ command -v ls
alias ls='ls --color=auto'
```

Reply:
This works: `sudo $(command -v docker) "$@"`. Updated.